### PR TITLE
Transaction tags overview

### PR DESCRIPTION
**Pull request by ncomerci**
image

### Tags

- `ESTATE DCL Marketplace Sales Fee`: Funds corresponding to the 2.5% fee applied to every ESTATE transaction (Minting or secondary)
- `LAND  DCL Marketplace Sales Fee`: Funds corresponding to the 2.5% fee applied to every LAND transaction (Minting or secondary)
- `NAME DCL Marketplace Sales Fee`: Funds corresponding to the 2.5% fee applied to every NAME transaction (Minting or secondary)
- `Wearable L1 Sales Fee`: Funds corresponding to the 2.5% fee applied to every Wearable transaction on Ethereum (Minting or secondary)
- `LooksRare Marketplace Fee`: Funds corresponding to the 2.5% fee applied to every transaction (ESTATE, LAND, NAME & Wearables) on LooksRare marketplace
- `OpenSea Marketplace Fee`: Funds corresponding to the 2.5% fee applied to every transaction (ESTATE, LAND, NAME & Wearables) on OpenSea marketplace
- `Wearable Submission Fee`: Funds corresponding to the fee applied to every new Wearable submission to the Decentraland Marketplace
- `Wearables Minting Fee`: Funds corresponding to the 2.5% fee applied to Wearables minting on Polygon network via the Decentraland Marketplace
- `DAO Committee`: Transactions between the DAO Treasury and the DAO Committee wallets (e.g. Transaction gas refunds)
- `Community Grants Payout`: Transactions corresponding to the funding of the vesting contracts for approved DAO Community Grants projects
- `Wearable Curators Committee Payout`: Transactions corresponding to the payout of compensations for members of the Wearables Curation Committee
- `Community Facilitation Payout`: Transactions corresponding to the payout for monthly compensations of the DAO Facilitator role
- `MANA Vesting Contract`: Funds corresponding to the 10-year MANA vesting contract that the DAO holds
- `Other`: Non-categorized or one-off transactions

### CODE REVIEW

File: src/components/Transparency/MonthlyTotal.tsx [hunk 1]
```diff
@@ -17,15 +18,31 @@ export type MonthlyTotalProps = React.HTMLAttributes<HTMLDivElement> & {
...
+
+const MAX_TAGS = 5
 
 export default React.memo(function MonthlyTotal({ title, monthlyTotal, invertDiffColors = false }: MonthlyTotalProps) {
   const t = useFormatMessage()
-  const [belowZeroColor, zeroOrOverColor] = invertDiffColors ? [GREEN, RED] : [RED, GREEN]
+  const [belowZeroColor, zeroOrOverColor] = invertDiffColors ? [Color.GREEN, Color.RED] : [Color.RED, Color.GREEN]
+  const [detailsVisibility, setDetailsVisibility] = useState(DetailsVisibility.OVERVIEW)
+
+  const click = () => {
```
Comment (andyesp) [inline src/components/Transparency/MonthlyTotal.tsx:38]: Rename this to `handleButtonClick` (or even more specific) so it's easier to see where this function is used 🙏

File: src/components/Transparency/MonthlyTotal.css [hunk 2]
```diff
@@ -3,6 +3,24 @@
...
+
+.MonthlyTotal--Overview {
+  overflow: hidden;
+  -webkit-transition: max-height 0.3s;
+  -moz-transition: max-height 0.3s;
+  transition: max-height 0.3s;
+}
+
+.MonthlyTotal.MonthlyTotal--Overview {
+  max-height: 398px;
```
Comment (andyesp) [inline src/components/Transparency/MonthlyTotal.css:21]: Can this propoerty be in the class above `.MonthlyTotal--Overview`?

Comment (ncomerci) [reply src/components/Transparency/MonthlyTotal.css:21]: Thanks, I was testing and forgot about it

**Review by andyesp — APPROVED** [review]

**Review by 1emu — APPROVED** [review]

**Comment by yemel** [comment]
Swap shouldn't be in the Income list

**Review by andyesp — APPROVED** [review]

**Review by 1emu — APPROVED** [review]

**Pull request merged**